mppm/troubleshoot/debug-mppm.md

```diff
@@ -6,6 +6,10 @@ description: Learn how to test a player running in Multiplayer Play Mode
 
 Use the Test Runner included in the [Unity Test Framework (UTF)](https://docs.unity3d.com/Packages/com.unity.test-framework@2.0/manual/index.html) to test players in a multiplayer project.
 
+:::note: 
+Multiplayer Play Mode is compatible with Unity Editor versions 2023.1 or later.
```

Comment:
Same question about 1.1.0/more generic as above.

Reply:
Just double-check with @kpurcellunity: 

Should we always specify versions 6000.0.3f1 or later in 1.1.0 docs?

Reply:
Yes we do need to be specific to the unity 6 version, I'll update this.